common/test/acceptance/tests/lms/test_lms_dashboard_search.py

```diff
@@ -0,0 +1,192 @@
+                'display_name': 'Test Course B '
+            },
+            'C' : {
+                'org': 'test_org',
+                'number': self.unique_id,
+                'run': 'test_run_C',
+                'display_name': 'Test Course C '
+            }
+        }
+
+        # generate course fixtures and outline pages
+        self.course_outlines = {}
+        self.course_fixtures = {}
+        for key, course_info in self.courses.iteritems():
+            course_outline = CourseOutlinePage(
+                self.browser,
+                course_info['org'],
+                course_info['number'],
+                course_info['run']
+            )
+
+            course_fix = CourseFixture(
+                course_info['org'],
+                course_info['number'],
+                course_info['run'],
+                course_info['display_name']
+            )
+
+            course_fix.add_children(
+                XBlockFixtureDesc('chapter', 'Section 1').add_children(
+                    XBlockFixtureDesc('sequential', 'Subsection 1').add_children(
+                        XBlockFixtureDesc('problem', 'dashboard search')
+                    )
+                )
+            ).add_children(
+                XBlockFixtureDesc('chapter', 'Section 2').add_children(
+                    XBlockFixtureDesc('sequential', 'Subsection 2')
+                )
+            ).install()
+
+            self.course_outlines[key] = course_outline
+            self.course_fixtures[key] = course_fix
+
+    def tearDown(self):
+        """
+        Remove index file
+        """
+        os.remove(self.TEST_INDEX_FILENAME)
+
+    def _auto_auth(self, username, email, staff):
+        """
+        Logout and login with given credentials.
+        """
+        LogoutPage(self.browser).visit()
+        AutoAuthPage(self.browser, username=username, email=email, staff=staff).visit()
+
+    def _studio_add_content(self, course_outline, html_content):
+        """
+        Add content on studio course page under specified section.
+        """
+        # create a unit in course outline
+        course_outline.visit()
+        subsection = course_outline.section_at(0).subsection_at(0)
+        subsection.toggle_expand()
+        subsection.add_unit()
+
+        # got to unit and create an HTML component and save (not publish)
+        unit_page = ContainerPage(self.browser, None)
+        unit_page.wait_for_page()
+        add_html_component(unit_page, 0)
+        unit_page.wait_for_element_presence('.edit-button', 'Edit button is visible')
+        click_css(unit_page, '.edit-button', 0, require_notification=False)
+        unit_page.wait_for_element_visibility('.modal-editor', 'Modal editor is visible')
+        type_in_codemirror(unit_page, 0, html_content)
+        click_css(unit_page, '.action-save', 0)
+
+    def _studio_publish_content(self, course_outline):
+        """
+        Publish content on studio course page under specified section.
```

Comment:
Seems to always be the first section - seems okay to me, but doc string is now out of date